introduction.md

```diff
@@ -0,0 +1,3 @@
+Hi ! I'm Nirina Andriamanantsara and I'm glad to follow the Mastering Github's course created by Codeschool. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.